web/src/main/resources/static/app.js

```diff
@@ -1,5 +1,4 @@
 const DEBUG = true
```

Comment:
Open question: arent't polluting the global namespace? Shouldn't we create a dedicated module (remember that I'm a JavaScript newbie)

Reply:
yes we are polluting. however the code base is small and we're already far from the state of the art js setup (which would be a total overkill with a completely separate project setup/packaging/linting/etc.. for what we're trying to do) so it's very acceptable as it is.